homeassistant/components/tuya/button.py

```diff
@@ -24,34 +24,79 @@
     "sd": (
         ButtonEntityDescription(
             key=DPCode.RESET_DUSTER_CLOTH,
+            action=DPCode.RESET_DUSTER_CLOTH,
+            action_value=True,
```

Comment:
`action_value` it not a member of `ButtonEntityDescription`

Reply:
I know, but I wanted to raise the possibility of using specific properties for the button entity.

Reply:
🤷 So this isn't a PR / Contribution?

Reply:
Sorry if I demonstrated in the wrong place. The PR will be closed. Thanks.